build.gradle

```diff
@@ -24,7 +24,8 @@ plugins {
     id "de.sebastianboegl.shadow.transformer.log4j" version "2.1.1"
     id "com.simonharrer.modernizer" version '1.5.0-1'
     id 'me.champeau.gradle.jmh' version '0.4.3'
-    id 'net.ltgt.errorprone' version '0.0.13'
+    id 'com.zyxist.chainsaw' version '0.1.3'
+    //id 'net.ltgt.errorprone' version '0.0.13'
```

Comment:
Can't we support error prone any more? This should be commented inside build.gradle

Reply:
at that time it did not work. Therefore I uncommented that. But have no clue if it's just the version or module stuff.